### PR TITLE
Refactored final-reject for pipeline integration

### PR DESCRIPTION
**Pull request by F-said**
- standardized return value for the pipeline loop
- standardized parameters

### CODE REVIEW

**Review by SDOsmany — COMMENTED** [review]
@F-said 
hey in line 29? did you mean to use autoRej.transform()  instead of fit_transform()

the fit_transform() method : Estimates the rejection params and finds bad epochs.

the transform() method : Removes bad epochs, repairs sensors, and returns clean epochs

**Comment by F-said** [comment]
Thanks for that catch @SDOsmany , keeping this PR open but making a new commit. Will re-request you again when fixed.

**Comment by F-said** [comment]
split fit_transform into fit() & transform()

**Review by SDOsmany — APPROVED** [review]
this looks great!

**Comment by georgebuzzell** [comment]
Thank you @F-said and @SDOsmany !!

File: base_eeg/test.py [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
     easy_montage = mne.channels.make_standard_montage('GSN-HydroCel-129')
     epochs.set_montage(easy_montage)
 
-    #Sample user parameter 
-    user_params={'tmin':-0.2,'tmax':0.5}
-    output_dict_finalRej=fr.final_reject_epoch(user_params, epochs, raw)
-    print('this is the the output dictionary ---> ',output_dict_finalRej)
+    # Sample user parameter
+    output_dict_finalRej = fr.final_reject_epoch(epochs)
+    print('this is the the output dictionary ---> ', output_dict_finalRej)
```
Comment (SDOsmany) [inline base_eeg/test.py:28]: @F-said i dont see any complaints for here. Could it be that it doesn't show it because the pull request is closed?

**Pull request merged**